StructureDefinition/England-PractitionerRole.xml

```diff
@@ -0,0 +1,353 @@
+      </constraint>
+      <constraint>
+        <key value="role-gmcreferencenumber" />
+        <severity value="error" />
+        <human value="GMC Reference Number must be of the format NNNNNNN" />
+        <expression value="identifier.where(system='https://fhir.hl7.org.uk/Id/gmc-reference-number').exists().not() or (identifier.where(system='https://fhir.hl7.org.uk/Id/gphc-number').exists()  and identifier.where(system='https://fhir.hl7.org.uk/Id/gmc-reference-number').value.matches('^[0-9]{7}$'))" />
+        <source value="https://fhir.nhs.uk/StructureDefinition/England-Practitioner" />
+      </constraint>
+      <constraint>
+        <key value="nacs-code-retired" />
+        <severity value="warning" />
+        <human value="NACS (/ODS) Practitioner Identifier is retired in NHS FHIR and should not be used. Please use the actual naming system instead (e,g, https://fhir.hl7.org.uk/Id/gmc-number, https://fhir.hl7.org.uk/Id/gmp-number, etc)." />
+        <expression value="identifier.where(system='https://fhir.hl7.org.uk/Id/professional-code').exists().not()" />
+        <source value="https://fhir.nhs.uk/StructureDefinition/England-Practitioner" />
+      </constraint>
+      <mustSupport value="true" />
+    </element>
+    <element id="PractitionerRole.organization">
+      <path value="PractitionerRole.organization" />
+      <definition value="This **MUST** either be a reference to a Organization resource or an identifier reference. In both cases the reference must contain an ANANA/ODS organisation code. &#xD;&#xA;&#xD;&#xA;&#xD;&#xA;#### organisation resource references&#xD;&#xA;&#xD;&#xA;It is recommended an *identifier reference* is included. &#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;organization&quot;: {&#xD;&#xA;        &quot;identifier&quot;: {&#xD;&#xA;            &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;            &quot;value&quot;: &quot;RBA&quot;&#xD;&#xA;          },&#xD;&#xA;        &quot;display&quot;: &quot;TAUNTON AND SOMERSET NHS FOUNDATION TRUST&quot;&#xD;&#xA;    }&#xD;&#xA;```&#xD;&#xA;&#xD;&#xA;Optionally *Resource reference* can be provided. This is required in the current version of EPS but this will be changed to supporting the *identifier reference* only.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&quot;organization&quot;: {&#xD;&#xA;        &quot;reference&quot;: &quot;urn:uuid:17c4270d-6966-4788-8cbc-1d1d63536b25&quot;,&#xD;&#xA;        &quot;identifier&quot;: {&#xD;&#xA;            &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;            &quot;value&quot;: &quot;RBA&quot;&#xD;&#xA;          },&#xD;&#xA;        &quot;display&quot;: &quot;TAUNTON AND SOMERSET NHS FOUNDATION TRUST&quot;&#xD;&#xA;    }&#xD;&#xA;```" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
```

Comment:
As per discussion, should not be adding references

Reply:
Pushed change